.circleci/config.yml

```diff
@@ -1215,3 +1217,10 @@ workflows:
 
       # Production releases.
       - deploy-and-release: *defaults_deploy
+      - deploy-docs:
+          requires:
+            - deploy-and-release
+          filters:
+            branches:
+              only: master
```

Comment:
Happy to change this for a branch check in each step for consistency with deploy-and-release